outbound.js

```diff
@@ -1144,7 +1147,12 @@ HMailItem.prototype.try_deliver_host_on_socket = function (mx, host, port, socke
         }
 
         // TLS
-        if (smtp_properties.tls && cfg.enable_tls && !secured) {
+        if (!(self.todo.domain in tls_config.no_tls_hosts) &&
+            !(host in tls_config.no_tls_hosts) &&
+            smtp_properties.tls && cfg.enable_tls && !secured)
+        {
+            console.log(tls_config.no_tls_hosts);
```

Comment:
One more important feature I need to add to this: inbound and outbound need to be able to specify separate cypher suites. You might want inbound to be much more strict than outbound.

> On Feb 22, 2016, at 4:42 PM, Matt <helpme@gmail.com> wrote:
> 
> Yeah it works, but obviously no console.logs should be there :)
> 
>> On Mon, Feb 22, 2016 at 4:15 PM, Matt Simerson <notifications@github.com> wrote:
>> In outbound.js:
>> 
>> > @@ -1144,7 +1147,12 @@ HMailItem.prototype.try_deliver_host_on_socket = function (mx, host, port, socke
>> >          }
>> >  
>> >          // TLS
>> > -        if (smtp_properties.tls && cfg.enable_tls && !secured) {
>> > +        if (!(self.todo.domain in tls_config.no_tls_hosts) &&
>> > +            !(host in tls_config.no_tls_hosts) &&
>> > +            smtp_properties.tls && cfg.enable_tls && !secured)
>> > +        {
>> > +            console.log(tls_config.no_tls_hosts);
>> Please read the messages. It sure looks like @baudehlo is still working on it. You'll also notice that it's in an -alpha release version, for early testing. Who was silly enough to claim production ready?
>> 
>> —
>> Reply to this email directly or view it on GitHub.
>> 
>